public/main.js

```diff
@@ -0,0 +1,37 @@
+const { app, BrowserWindow } = require('electron');
+const isDev = require('electron-is-dev');
+
+if (isDev) {
+  console.log('Running in development');
+} else {
+  console.log('Running in production');
+}
+
+let mainWindow;
+
+function createWindow() {
+  mainWindow = new BrowserWindow({
+    width: 800,
+    height: 600,
+    webPreferences: {
+      nodeIntegration: true,
+    },
+  });
+
+  // mainWindow.loadURL(isDev ? 'http://localhost:3000' : `file://${__dirname}/../../public/index.html`);
+  mainWindow.loadURL(`file://${__dirname}/index.html`);
+
+  mainWindow.on('closed', () => {
+    mainWindow = null;
+  });
+}
+
+app.on('ready', createWindow);
+
+app.on('window-all-closed', () => {
+  if (process.platform !== 'darwin') app.quit();
```

Comment:
@Donaldblanc, did you figure out if this kills the main process? Or is that still something we need to implement in some other way?

Reply:
this currently still does not kill the process, it was from the template Joe initially configured in the react room. I will create an issue for this to pick up during the next sprint.